website/docs/docs/core/connect-data-platform/profiles.yml.md

```diff
@@ -31,6 +31,10 @@ This section identifies the parts of your `profiles.yml` that aren't specific to
   [fail_fast](/reference/global-configs/failing-fast): <true | false>
   [use_experimental_parser](/reference/global-configs/parsing): <true | false>
   [static_parser](/reference/global-configs/parsing): <true | false>
+  [cache_selected_only](/reference/global-configs/cache): <true | false>
+  [no_print](/reference/global-configs/print-output#suppress-print-messages-in-stdout): <true | false>
+  [target-path](/reference/global-configs/logs#log-and-target-paths): "other-target"
+  [log-path](/reference/global-configs/logs#log-and-target-paths): "other-logs"
```

Comment:
⚠️ Oops! I made a typo [here](https://github.com/dbt-labs/docs.getdbt.com/issues/3264) that also got copied to [here](https://dbt-labs.slack.com/archives/C03SK9W08BC/p1705112397826119?thread_ts=1705063983.007889&cid=C03SK9W08BC). Sorry 'bout that!

### What we should do

1. Since `target-path` and `log-path` are _only_ applicable to `dbt_project.yml`, we should remove them here.
2. Since `printer_width` and `log_format` from https://github.com/dbt-labs/docs.getdbt.com/pull/4755 _are_ applicable to `profiles.yml`, we should add them instead:

```suggestion
  [printer_width](/reference/global-configs/print-output#printer-width): <integer>
  [log_format](/reference/global-configs/logs): <text | json | default>
```